subliminal/providers/subscenter.py

```diff
@@ -201,7 +203,7 @@ def query(self, title, season=None, episode=None):
 
                         # otherwise create it
                         subtitle = SubsCenterSubtitle(language, hearing_impaired, page_link, title, season, episode,
-                                                      title, subtitle_id, subtitle_key, downloaded, [release])
+                                                      title, subtitle_id, subtitle_key, downloaded, [release], h_version)
```

Comment:
`h_version` is an identifier for the subtitle provided by the provider, see example : 
http://www.subscenter.org/he/cst/data/series/sb/greys-anatomy/13/22/